tutorials/receiving-money.md

```diff
@@ -0,0 +1,275 @@
+
+##### Who bears Paystack transaction fees
+
+Finally there is the question of who bears Paystack transaction fees for split payment transactions. By default, Paystack takes the transaction fee from the main account’s share. (Paystack transaction fee is 1.5% of the amount below N2500 and 1.5% + N100 for transactions above N2500). In this N2000 sample transaction, Paystack fee will be N30. 
+
+So by default, using the split percentage for our sample transaction, subaccount will get N1800, main account will get N170 and Paystack will get N30
+
+If you want the sub-account to bear the fees, you can specify that in the initialization object like so: bearer: 'subaccount'.  In this case, subaccount gets N1770, main account gets N200 and Paystack gets N30.
+
+Whichever account is set to bear the Paystack fees, if the fees is greater than the amount meant for the bearer, Paystack takes the fees from the other account. For example, in our sample N2000 transaction, assuming the split percentage is 1% this means that the main account should get N20 and subaccount gets N1980.
+
+But Paystack transaction fee is N30. So if the main Paystack account is supposed to bear the transaction fees, the fees are greater than the amount meant for the main account so we take the transaction fee from the subaccount instead. So Paystack gets N30, main account gets N20, and subaccount gets N1950.
+
+So a typical split payment transaction initialization object will look something:
+
+```
+{
+  "email": "customer@email.com",
+  "amount": 200000,
+  "subaccount": "ACCT_4hl4xenwpjy5wb",
+  "transaction_charge": 30000,
+  "bearer": "subaccount"
+}
+```
+
+####Settlement
+
+Paystack settles subaccounts automatically everyday into the settlement bank account provided when creating the subaccounts. The amount for the main accounts are settled according to the settlement schedule set on the settlement account (Next day settlement/manual settlement).
+
+####Best Practices
+
+Subaccounts are mostly used by companies that have separate branches/outlets and would like to know which transactions are coming from which outlets. It is not compulsory to split the payments, you can set the split percentage to pay 100% of the payment into the main account, and still use subaccount codes to differentiate the payments.
+
+You cannot use subaccounts created in test mode with your live keys and vice versa.
+
+###Use Case
+
+## Recurring Payments
+
+Paystack allows you to charge a card subsequently using our APIs such the user doesn’t need the enter the card details again after a first successful payment. This allows you to debit the user automatically for subcriptions, in-app payments, thrift collections, or simply charge returning customers. A common misconception is that you need to collect and tokenize user’s card details in order to do this. You don’t need to do that. In this section we will explain how to get and use Paystack card authorization tokens.
```

Comment:
Paystack allows you to charge a card subsequently using our APIs such **_that_** the user doesn’t need _**to**_ enter the card details again after a first successful payment